packages/components/charts/src/use-pie-chart.ts

```diff
@@ -218,9 +227,10 @@ export const usePieChart = ({
         labelOffset,
         styles: styles.label,
         labelProps,
+        percentProps: percent,
```

Comment:
Change `percentProps: percent` to `percent` and remove `isPercent`.

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3609/files?w=1#diff-950161e6b78e396d2ab4b4c08a3c65958e1b5ed57f0d9547ca0e20eeaef3044eR231